### PR TITLE
Fix internal redirect url.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@miltontony & @Rizziepit ready for review.

File: unicoremc/views.py [hunk 1]
```diff
@@ -259,15 +261,17 @@ class AppEventSourceView(ProjectViewMixin, View):
 
     def get(self, request, project_id, task_id, path):
         project = get_object_or_404(Project, pk=project_id)
```
Comment (Rizziepit) [inline unicoremc/views.py:265]: Can you use `self.get_queryset()` here like in `AppLogView` [here](https://github.com/universalcore/unicore-mc/blob/feature/issue-120-fix-redirect/unicoremc/views.py#L249)? Should I open an issue? It limits projects to ones the user has permission to access. Sorry, I didn't review the PR that made that change.

**Comment by Rizziepit** [comment]
+1. Permission issue doesn't have to be fixed here

File: unicoremc/views.py (hunk 1 above)
Comment (smn) [inline unicoremc/views.py:265]: I'll change it in this PR.

Comment (smn) [inline unicoremc/views.py:265]: Hm nope, doing a separate PR.

**Pull request merged**